server/chat.ts

```diff
@@ -1344,6 +1344,15 @@ export class CommandContext extends MessageContext {
 		}
 		return this.room;
 	}
+	// eslint-disable-next-line @typescript-eslint/type-annotation-spacing
+	getGame<T extends RoomGame>(constructor: new (...args: any[]) => T) {
+		const room = this.requireRoom();
+		if (!room.game) throw new Chat.ErrorMessage(`There is no game going on in this room.`);
+		const game = room.getGame(constructor);
+		// must be a different game
+		if (!game) throw new Chat.ErrorMessage(`There is already a game of ${room.game.title} going on.`);
```

Comment:
These error messages should probably mention what game _should_ be going on. `constructor.name` would do this, but we could add a static property like `gameName` if we wanted something nicer.

Reply:
```suggestion
		if (!room.game) throw new Chat.ErrorMessage(`This command requires a game of ${constructor.name} (this room has no game).`);
		const game = room.getGame(constructor);
		// must be a different game
		if (!game) throw new Chat.ErrorMessage(`This command requires a game of ${constructor.name} (this game is ${room.game.title}).`);
```